app/browser/reducers/tabsReducer.js

```diff
@@ -67,6 +67,11 @@ const updateActiveTab = (state, closeTabId) => {
     }
   }
 
+  // fall back to first tab
+  if (nextTabId === tabState.TAB_ID_NONE) {
+    nextTabId = tabState.get('tabs').first()
+  }
+
   if (nextTabId !== tabState.TAB_ID_NONE) {
```

Comment:
When I wrote the unit tests for this, I was unable to find a way to hit this condition.  When you originally pulled the logic out of frameStateUtil and moved here, you had a similar block.  Since it wasn't possible to hit, I had removed it.  Did you see a case where this runs?